docs/commands/browser/$$.mdx

```diff
@@ -79,9 +84,9 @@ it("should get text a menu link - JS Function", async ({ browser }) => {
 
 ## Связанные команды {#related}
 
--   [browser.$](./$)
--   [element.$](../element/$)
--   [element.$$](../element/$$)
+-   [browser.$](../_dollar)
+-   [element.$](../../element/_dollar)
```

Comment:
can you, pls, explain how it works?

Reply:
With `trailingSlash` set to `true` every page always has a slash in the end. For instance, `docs/v8/quickstart` -> `docs/v8/quickstart/`. This means that for relative paths, current page is a directory. So if we are on `docs/v8/commands/browser/_dollardollar/` page, to reference a page on a same level we still need to go back because it's a directory in terms of url.